components/Mappings/Mappings.js

```diff
@@ -753,7 +760,7 @@ class Mappings extends Component {
 
 							{this.state.editable ? (
 								<Button type="primary" onClick={this.handleSynonymModal}>
-									{this.state.synonyms ? 'Edit Synonym' : 'Add Synonym'}
+									{`${this.state.synonyms ? 'Edit' : 'Add'} Synonyms`}
```

Comment:
Please revert it back, since antd buttons add spans for its children and they get joined together. Check the UI before making such changes in the future.

Reply:
cc @jyash97.

Reply:
Okay Checking